### DISCUSSION TITLE
get number of queries being refetched / revalidated like useIsFetching

### DISCUSSION THREAD

**Discussion by siddiquiaffan** [TanStack/query#6135] [Q&A]
Is there any way we can get number of queries being refetched / revalidated like useIsFetching.

**Comment by Lisba** [comment]
Hi, @siddiquiaffan! You can use the [queryClient.getQueryState()](https://tanstack.com/query/v4/docs/react/reference/QueryClient#queryclientsetquerydata) with a [predicate as a filter](https://tanstack.com/query/v4/docs/react/guides/filters#query-filters) to analyze the state of your queries and get the ones that fits with your requirements, here you have the [state's props available](https://tanstack.com/query/v5/docs/react/reference/useQuery) to evaluate.

**Reply by siddiquiaffan** [reply]
I want to subscribe to that value, like we do using ```useIsFetching()```.

**Comment by TkDodo** [comment]
> I want to subscribe to that value

isn't that exactly what `useIsFetching` is doing?
